### PR TITLE
show different forms for developer and product team

### PR DESCRIPTION
**Pull request by MounikaReddy15**
## Purpose
_Redirect users based on whether they are developer or product team during user registration_

## Further info
_showing survey for developer and new product form for product team_

## Ticket number
#66

### CODE REVIEW

File: src/pages/Register/Register.js [hunk 1]
```diff
@@ -288,6 +291,35 @@ const Register = ({
                     />
                   </Grid>
                 </Grid>
+                <Grid container spacing={isMobile() ? 0 : 3}>
```
Comment (RadhikaPPatel) [inline src/pages/Register/Register.js:294]: Why is this autocomplete field? It needs to be a normal select field only.

File: src/pages/Dashboard/components/UserDashboard.js [hunk 2]
```diff
@@ -254,7 +254,7 @@ const UserDashboard = (props) => {
               ))}
           </TextField>
         </Grid>
-        <Grid item xs={4} md={3} lg={2} textAlign="end">
+        {/* <Grid item xs={4} md={3} lg={2} textAlign="end">
```
Comment (RadhikaPPatel) [inline src/pages/Dashboard/components/UserDashboard.js:257]: Why is this hidden?

File: src/pages/Register/Register.js (hunk 1 above)
Comment (RadhikaPPatel) [reply src/pages/Register/Register.js:294]: Also, fix all the problems for this file.

**Review by RadhikaPPatel — CHANGES_REQUESTED** [review]
Please handle the requested changes

**Review by yasmin2496 — CHANGES_REQUESTED** [review]
@MounikaReddy15 Two things to check
1. For developers, after filling the survey it takes them back to the form page and the form is still visible. Ideally, this should hide once the form is filled
2. For product team, after successful filling up of form it stays on modal take the user to dashboard and close the modal. Also the alert message is incorrect

**Comment by yasmin2496** [comment]
@MounikaReddy15 There is also some problem with product modal now. After clicking on create product I can't close the modal being sure that my changes are saved. Check this as well

**Review by yasmin2496 — APPROVED** [review]

**Pull request merged**